scripts/lib/k8s.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# check_ds_rollout watches the status of the latest rollout until it's done or
+# until the timeout. Namespace and timeout are optional parameters
+check_ds_rollout() {
+    local __ds_name=${1:-}
+    local __namespace=${2:-}
+    local __timeout=${3:-"2m"}
+    local __args=""
+    if [ -n "$__namespace" ]; then
+        __args="$__args-n $__namespace"
+    fi
+    kubectl rollout status ds/"$__ds_name" $__args --timeout=$__timeout
```

Comment:
Can we add an error if rollout fails?

Reply:
kubectl will print error if timed out, eg: error: timed out waiting for the condition

Reply:
Sounds good.